src/styles/blocks/header.scss

```diff
@@ -0,0 +1,60 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  background-color: $color-white;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
```

Comment:
move each bem block to separate file

Reply:
Fix it 
![image](https://user-images.githubusercontent.com/97547824/199812543-25e1cb8f-6e6b-4be0-9bba-3a198709a7b6.png)